README.md

```diff
@@ -17,6 +17,16 @@ interactive and automatic proofs (see
 Prerequisites
 --------------
 
+* Glasgow Haskell Compiler ([GHC](https://www.haskell.org/ghc/))
+
+  We support the following versions of GHC: 7.6.3, 7.8.4, 7.10.3, 8.0.1, and 8.0.2.
```

Comment:
Better: Apia supports the versions of GHC supported by Agda, i.e. 7.6.3, 7.8.4, 7.10.3 and 8.0.2.